client/src/legacy/ElementEditor/entwine.js

```diff
@@ -28,6 +31,15 @@ jQuery.entwine('ss', ($) => {
 
     onunmatch() {
       ReactDOM.unmountComponentAtNode(this[0]);
-    }
+    },
+
+    /**
+     * Invalidate cache after the form is submitted to force apollo to re-fetch.
+     */
+    'from .cms-edit-form': {
```

Comment:
Typo? `form`?

Reply:
I thought so too buuuut I copied from admin - it appears this is some entwine magic - as in if this event comes __from__ `.cms-edit-form`.

Reply:
👍 TIL